### ISSUE TITLE
Unhandled session token refresh error: could not look up session at path

### ISSUE THREAD

**Issue by kspangsege** [realm/realm-js#1393]
Circa 10 minutes after opening a synchronized Realm, I see this error,

    Unhandled session token refresh error: could not look up session at path /home/kristian/js_test/realm-object-server/c97f762d-7660-43c2-b5ae-04931eb49fae/realm%3A%2F%2Flocalhost%3A9080%2F%7E%2Ffoo

This seems to happen when I specify a custom local file system path for the Realm.

The error message seems to indicate that the default path is incorrectly used. It should probably have used the custom path that I specified.

I suppose this issue could be related to https://github.com/realm/realm-js/issues/1392.

Full output:
```
Opening Realm file: /home/kristian/js_test/realm-object-server/c97f762d-7660-43c2-b5ae-04931eb49fae/realm%3A%2F%2Flocalhost%3A9080%2F%7E%2Ffoo
Connection[1]: Resolving 'localhost:9080'
Connection[1]: Connecting to endpoint '127.0.0.1:9080' (1/1)
Connection[1]: Connected to endpoint '127.0.0.1:9080' (from '127.0.0.1:35166')
Closing Realm file: /home/kristian/js_test/realm-object-server/c97f762d-7660-43c2-b5ae-04931eb49fae/realm%3A%2F%2Flocalhost%3A9080%2F%7E%2Ffoo
Wait 10 minutes
Opening Realm file: /home/kristian/js_test/foo.realm
Connection[2]: Resolving 'localhost:9080'
Connection[2]: Connecting to endpoint '127.0.0.1:9080' (1/1)
Connection[2]: Connected to endpoint '127.0.0.1:9080' (from '127.0.0.1:35174')
Unhandled session token refresh error: could not look up session at path /home/kristian/js_test/realm-object-server/c97f762d-7660-43c2-b5ae-04931eb49fae/realm%3A%2F%2Flocalhost%3A9080%2F%7E%2Ffoo
```

The script:
```javascript
const Realm = require('realm');

Realm.Sync.User.login('http://localhost:9080', 'test', 'test').then(user => {
    Realm.open({
        path: 'foo.realm',
        sync: {
            user: user,
            url:  'realm://localhost:9080/~/foo',
        },
        schema: [],
    }).then(realm => {
        console.log('Wait 10 minutes');
    });
});
```

**Comment by kspangsege** [comment]
Additional observation: If I remove the custom path specification, the error seems to no longer occur.

**Comment by bdash** [comment]
Fixed by #1402.
